packages/frontend/package.json

```diff
@@ -63,7 +64,7 @@
   },
   "scripts": {
     "build": "craco build",
-    "start": "TAILWIND_MODE=watch craco start",
+    "start": "PORT=$(grep FRONTEND_DEVELOPMENT_PORT ../../.env | cut -d '=' -f2) && TAILWIND_MODE=watch env-cmd -f ../../.env craco start",
```

Comment:
We should not override env variables in command except for NODE_ENV.  Also this is a crazy way to manually parse an env file that should not be necessary.

Reply:
Ok. Slightly hacky, agreed. :) This came from my want to join all ENV in one file. 

We could move some stuff back to package individual env files. But, then we need to find a solution to this problem (`env-cmd` I believe only supports reading env from one file):

1. Read top level .env
2. Read package level .env
3. Merge and feed env to package on start

Reply:
It looks like you can just call `env-cmd` twice to merge two files: 

```
    "start": "TAILWIND_MODE=watch env-cmd -f ../../.env env-cmd -f .env --silent craco start",
```

From this comment: https://github.com/toddbluhm/env-cmd/issues/148#issuecomment-652807598